packages/contracts/contracts/L1/SystemConfig.sol

```diff
@@ -130,7 +143,8 @@ contract SystemConfig is OwnableUpgradeable, Semver {
         bytes32 _batcherHash,
         uint64 _gasLimit,
         address _unsafeBlockSigner,
-        ResourceMetering.ResourceConfig memory _config
+        ResourceMetering.ResourceConfig memory _config,
```

Comment:
If we take the validatorRewardScalar value to be uint64, I guess we could use the slot with gasLimit.
Or maybe leave the 32 bytes unused in case we want to add other config values in the future?

Reply:
It's the same concept as the `_scalar` above.
And it's already deployed and used in the L1Block contract as a uint256 type, so it can't be changed to a different type for type matching.